osc/build.py

```diff
@@ -1320,6 +1322,12 @@ def __str__(self):
                         print("Using prefered package: " + path + "/" + filename)
                         os.unlink(tffn)
 
+        if bi.containerannotation:
+            if not os.path.exists("containers"):
+                os.makedirs("containers")
```

Comment:
Replace with `os.makedirs("containers", exist_ok=True)`.
Your code is longer and is prone to race conditions.